app/controllers/api/v2/organisations_controller.rb

```diff
@@ -2,16 +2,24 @@ module API
   module V2
     class OrganisationsController < API::V2::ApplicationController
       def index
+        per_page = params[:page][:per_page] || 10
+
         authorize Organisation
         @organisations = Organisation.all
         if params_includes_provider?
           current_recruitment_cycle = RecruitmentCycle.current
           @organisations = @organisations.includes(:providers, :users, :nctl_organisations)
                              .where(provider: { recruitment_cycle_id: current_recruitment_cycle.id })
+
         end
 
-        render jsonapi: @organisations, include: params[:include],
-          fields: { providers: %i[provider_code provider_name], users: %i[email sign_in_user_id first_name last_name] }
+        render jsonapi: paginate(@organisations.order(:name), per_page: per_page),
```

Comment:
I notice that we have an N+1 query in here ...

```
2020-06-24 21:10:19.163050 D [96597:puma threadpool 002] (0.821ms) ActiveRecord -- { :sql => "SELECT COUNT(*) FROM \"course\" WHERE \"course\".\"provider_id\" = $1 AND \"course\".\"discarded_at\" IS NULL", :binds => { :provider_id => 13996 } }
2020-06-24 21:10:19.164485 D [96597:puma threadpool 002] (0.414ms) ActiveRecord -- { :sql => "SELECT COUNT(*) FROM \"course\" WHERE \"course\".\"provider_id\" = $1 AND \"course\".\"discarded_at\" IS NULL", :binds => { :provider_id => 13329 } }
2020-06-24 21:10:19.165423 D [96597:puma threadpool 002] (0.307ms) ActiveRecord -- { :sql => "SELECT COUNT(*) FROM \"course\" WHERE \"course\".\"provider_id\" = $1 AND \"course\".\"discarded_at\" IS NULL", :binds => { :provider_id => 14190 } }
...
```

one for each provider. Paging is helping out, but there's also [`Provider.include_course_count`](https://github.com/DFE-Digital/teacher-training-api/blob/master/app/models/provider.rb#L157) which should optimise this, if it'll work in this query. Is it worth adding it in while we're making changes here?

Reply:
That's coming from the `meta` block in the `SerializableProvider`. I think we need to get rid of that which is probably beyond the scope of this PR. Did you have a way of hooking in `Provider.include_course_count`?